plugins/orchestrator-backend/src/service/SonataFlowService.test.ts

```diff
@@ -203,68 +187,57 @@ describe('SonataFlowService', () => {
       );
       expect(result).toEqual({ id: definitionId, status: 'completed' });
     });
-    it('should log an error and return undefined when the fetch response is not ok without extra info', async () => {
+    it('should propagate thrown error when the fetch response is not ok without extra info', async () => {
       // When
-      const mockResponse = setupTest({
+      setupTest({
         ok: false,
         status: 500,
         statusText: 'Internal Server Error',
         json: { details: undefined, stack: undefined },
       });
 
-      const result = await runErrorTest();
+      let result;
+      try {
+        await runErrorTest();
+      } catch (error) {
+        result = error;
+      }
```

Comment:
same question about `.rejects`

Reply:
it works so what's wrong with the current code?

Reply:
it works so what's wrong with the current code?